mods/cnc/weapons/missiles.yaml

```diff
@@ -234,6 +243,7 @@ TowerMissile:
 		RangeLimit: 8c409
 	Warhead@1Dam: SpreadDamage
 		Spread: 483
+		Damage: 3000
 		ValidTargets: Ground
 		Versus:
 			None: 52
```

Comment:
Seeing that this inherits `Concrete: 140` from `^MissileWeapon`, doesn't this then need to define `Concrete: 100` to match the heavy armor?